### PR TITLE
Add Numpy Frontend trunc

### LINKED ISSUE #3503: trunc
Implement Numpy's trunc function. One of the tasks present in #1525

### PR DESCRIPTION
**Pull request by aryanxk02**
Fixes #3503

### CODE REVIEW

**Comment by aryanxk02** [comment]
Hey @CatB1t,

I wrote code for implementing trunc function, but the commit shows commits from my previous PR where I have implemented cbrt function. 

Why is it so?

**Comment by aryanxk02** [comment]
Okay, I have resolved the merge conflicts!

**Comment by CatB1t** [comment]
Hello @aryanxk02, Please add the test for the function `trunc`.

**Comment by aryanxk02** [comment]
Hello @CatB1t, I have added the test function for the ```trunc``` function. Please review!

**Comment by MrZilinXiao** [comment]
> Hello @CatB1t, I have added the test function for the `trunc` function. Please review!

Hi @aryanxk02, the test you added recently has not passed numpy frontend testing. Please consider reviewing your code :)
https://github.com/unifyai/ivy/runs/7988315371?check_suite_focus=true#step:4:4363

**Comment by aryanxk02** [comment]
Hello @MrZilinXiao how do I approach exactly? Can you give me a few pointers on what changes must be done in order to get the tests passing? In this case numpy frontend tests!

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_arithmetic_operations.py [hunk 1]
```diff
@@ -397,3 +397,53 @@ def test_numpy_cbrt(
         subok=True,
         test_values=False,
     )
+
+
+# trunc
```
Comment (CatB1t) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_arithmetic_operations.py:402]: Please add the @handle_cmd_line_args, this allows us to handle with_out, as_variable, etc, by us.

**Review by CatB1t — CHANGES_REQUESTED** [review]

**Comment by aryanxk02** [comment]
Do I have to replace the code written from line 403 to line 413 and replace with ```@handle_cmd_line_args``` @CatB1t

**Comment by aryanxk02** [comment]
Do I have to remove the contents present from line 403 to line 413 and add ```@handle_cmd_line_args```? @CatB1t

**Comment by CatB1t** [comment]
Hello @aryanxk02, sorry for the confusion, these parameters shouldn't be generated ey you: 
- `with_out`
- `native_array`
- `where`
- `as_variable`

and therefore you should remove them, after adding the `@handle_cmd_line_args` decorator.

**Comment by aryanxk02** [comment]
The requested changes have been made @CatB1t. Please review and let me know if anything needs to be changed!

**Comment by CatB1t** [comment]
Hello @aryanxk02, the tests for `trunc` is not passing, please fix that. https://github.com/unifyai/ivy/runs/7998192463?check_suite_focus=true#step:4:4774

**Comment by MrZilinXiao** [comment]
Hi all! I am responsible for checking stale PRs regularly. Since this PR has been inactive for 30 days, I am wondering if this PR is still ongoing. I will now close it. In the meanwhile, you could reopen it if you believe the PR is still useful for you.

**Pull request closed without merging**